app/src/test/java/com/github/barriosnahuel/vossosunboton/ui/home/PlaybackClickListenerTest.kt

```diff
@@ -0,0 +1,66 @@
+package com.github.barriosnahuel.vossosunboton.ui.home
+
+import android.view.View
+import android.widget.ToggleButton
+import com.github.barriosnahuel.vossosunboton.feature.playback.PlayerControllerFactory
+import com.github.barriosnahuel.vossosunboton.model.Sound
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.mockkObject
+import io.mockk.verify
+import org.junit.Test
+
+class PlaybackClickListenerTest {
+
+    @Test
+    fun `onClick on a view that is not pressed should start an audio button`() {
+        val button = givenARecentCheckedView()
+
+        whenClickingOn(button)
+
+        thenItShouldStartPlayingAnAudio()
+    }
+
+    @Test
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [UndocumentedPublicFunction - [onClick on a view that is currently pressed should stop the audio button] at /src/app/src/test/java/com/github/barriosnahuel/vossosunboton/ui/home/PlaybackClickListenerTest.kt:24:5](https://app.codacy.com/app/barrios.nahuel/push-me/pullRequest?prid=4481303)